cachito/workers/tasks/utils.py

```diff
@@ -233,6 +234,26 @@ def set_packages_and_deps_counts(request_id: int, packages_count: int, dependenc
     )
 
 
+def sort_packages_and_deps_in_place(packages):
+    """
+    Sorts lists of packages in place.
+
+    Sorting order: name -> version -> type.
```

Comment:
This is now `type -> dev -> name -> version`

Reply:
Done!